recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
Thanks both - I have removed all references to TCLTKVERSION in the build.sh file; seems we can do without. That's a nice simplification.